### ISSUE TITLE
Tempo chart 1.6.12 won't install 

### ISSUE THREAD

**Issue by mateuszdrab** [grafana/helm-charts#2723]
Hi guys

Just a quick report that latest `tempo-distributed` chart won't deploy with version 1.6.12 due to error:

`executing "tempo-distributed/templates/gateway/secret-gateway.yaml" at <.Release.Namespace>: nil pointer evaluating interface {}.Namespace`

Rolling back to 1.6.11 works just fine.

Think it's something to do with https://github.com/grafana/helm-charts/commit/124b85bce60c274dee0f353100e68fe535a28ef5

Syntax of the namespace is `namespace: {{ .Release.Namespace }}` but other manifests in the chart have 'namespace: {{ .Release.Namespace | quote }}'

**Comment by kaflake** [comment]
I have the same problem with 1.6.13

**Comment by martensson** [comment]
Having the same issue with 1.7.0 today. Was not able to upgrade and got the exact same error.

**Comment by kaflake** [comment]
Okay, I have made a PullRequest, but was the first time this here and in GitHub. Hope this is working now. Reset the Version in the PullRequest noch on 1.7.1. Before I make 1.6.14 but I think I forgot to request a new PullRequest.

**Comment by michaelniemand** [comment]
same issue with 1.7.1 today.

**Comment by rknightion** [comment]
Does anyone have a viable workaround to get the chart to install or some kind of override we can provide perhaps?

**Comment by kaflake** [comment]
No, I just use the old chart. Waiting for the merge of the changes I make.
